lib/systems/include/Controllers/SimpleLaunchController.h

```diff
@@ -19,7 +19,7 @@ class TorqueControllerSimpleLaunch : public virtual BaseLaunchController
 public:
     /*!
         SIMPLE LAUNCH CONTROLLER
-        This launch controller is based off of a specified launch rate and an initial speed target
+        @brief launch controller is based off of a specified launch rate and an initial speed target
```

Comment:
full sentence please

Reply:
Emphasize that this simple launch controller does NOT take feedback, and simply targets an RPM according to a linear increase in velocity. If that's NOT how this works, and I'm misreading it... then your documentation is not sufficiently clear.